java-manta-client-unshaded/src/main/java/com/joyent/manta/exception/MantaErrorCode.java

```diff
@@ -180,4 +522,31 @@ public static MantaErrorCode valueOfCode(final Object object) {
 
         return valueOfCode(object.toString());
     }
+
+    /**
+     * Populates an Inclusive Error Map for {@link MantaErrorCode} by its code value.
+     *
+     * @return a thread-safe Manta Errors HashMap.
+     * @throws MantaClientException if a thread-safe access is not granted for Error Map.
+     */
+    public static Map<String, MantaErrorCode> createErrorMap() throws MantaClientException {
+        errorMap = Collections.synchronizedMap(new HashMap<>(values().length));
+        for (MantaErrorCode m : values()) {
+            try {
+                Field field = MantaErrorCode.class.getField(m.name());
```

Comment:
Since we no longer have duplicated error codes, we no longer need to skip loading deprecated values. Something simple that validates the contract not to duplicate error codes like the following should be fine:
```java
    public static Map<String, MantaErrorCode> createErrorMap() throws MantaClientException {
        final Map<String, MantaErrorCode> map = new LinkedHashMap<>(values().length);

        for (MantaErrorCode errorCode : values()) {
            final MantaErrorCode replaced = map.put(errorCode.code, errorCode);

            if (replaced != null) {
                String msg = String.format("Duplicate error code specified in enum: %s",
                        replaced.code);
                throw new IllegalArgumentException(msg);
            }
        }

        return map;
    }
```

Reply:
Noted.